config/initializers/rack_attack.rb

```diff
@@ -9,6 +9,8 @@ def routed_to
     "#{route[:controller]}##{route[:action]}"
   rescue ActionController::RoutingError
     nil
+  rescue NoMethodError
+    nil
```

Comment:
This is an ugly fix, but I've looked into the source code and couldn't find a way around this :(